proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -374,7 +374,7 @@ message LiquidityTierUpsertEventV1 {
   // The maximum position size at which the margin requirements are
```

Comment:
Might be helpful to note this is deprecated since some version in the comment, similar to Cosmos SDK [style](https://github.com/search?q=org%3Adydxprotocol+%22%2F%2F+Deprecated+post+v0.46.x%22&type=code)

I think here we should say `Deprecated since v3.x`?